### ISSUE TITLE
blades tables

### ISSUE THREAD

**Issue by cristianmtr** [cristianmtr/rpg_tables#14]
```
// Created by Odog4ever

streetmood
	Dark
	Cold
	Bright
	Lively
	Quiet
	Refined
	Abandoned 
	Decrepit 
	Cramped 
	Noisy
	Cozy
	Warm
	Dim
	//Moonlit ^0.1
	Fog-shrouded
	Damp
	Chill
	Shadowy
	Soot-choked
	Wind-swept
	Rain-slick
	Oil-slick
	
streettype
	Narrow Lane 
	Tight Alley 
	Twisting Street 
	Rough Road 
	Bridge 
	Waterway
	Closed Court 
	Open Plaza 
	Paved Avenue 
	Tunnel
	Wide Boulevard
	Roundabout
	Elevated Street
	Flooded Street
	Suspended Street
	Subterranean Street
	Floating Street
	{Private|Gated} Street
	
streetuse
	Residential Housing
	Craft Goods 
	Labor 
	Shops 
	Trade 
	Hospitality
	{the Law|Government}
	Public Spaces 
	Power 
	Manufacturing 
	Transportation 
	Leisure
	Vices 
	Entertainment 
	Storage 
	//Cultivation 
	Academia 
	Art
	
	
streetdetails
	Metal Supports
	Ironwork {Gates|Fences}
	Belching Chimneys
	Metal {Grates|Hatches|Doors}
	Clockwork Mechanisms
	Exposed {Rigging|Cables}
	{Stairs|Ramps|Terraces}
	Wooden Scaffolds
	Skyways
	Rooftop Spaces
	{Train Rails|Train Cars}
	Hidden Passages
	{Banners|Pennants}
	Festival Decorations
	{Crowds|Rioting}
	Street Performers
	{Makeshift Stalls|Shelters}
	Crisscrossing Routes
	Gang Markings
	Patrol Posts 
	Lookouts
	Stocks for Public Punishment
	{Street Criers|Street Preachers}
	{News Stands|Public Notices}
	Stray Animals 
	Landscaping 
	Muck
	//{Construction|Demolition}
	{Foul Runoff|Fumes|Smoke}
	{Orphans|Beggars}
	Ancient Ruins
	Leering Gargoyles
	{Spirit Chimes|Wards}
	Eerie Emptiness
	Recent {Quarantine|Lockdown}
	Shrine Offerings
	{Belltower|Clocktower}

MRstreetfeatures
	Arcade
	Awnings
	Balconies
	Barricades
	Bridges
	Canal
	Carriages
	Catwalks
	Clothslines
	Crowds
	Dead End
	Unnaturally dense fog
	Flooding
	Food Stalls
	Fountain
	Gates
	Ladders
	Livestock
	Mud
	Roof access
	Roof gardens
	Sewer access
	Sinkhole
	Steep roofs
	Steps
	Vermin
	Wells

		
streetsights
	Dancing Shadows 
	{Flickering Lights|Gas-lights|Electrolights|Lanterns}
	{Mist|Fog} 
	Echoes in the Ghost Field
	{Soot|Ash Clouds|Grime}
	{Crackling Electric Wires|Crackling Electric Mechanisms} 
	Oily puddles
	{Fountain|Cistern|Grotto|Park} 
	Statue
	Radiant Energy Garden
	Walkways
	Brick Walls
	Cobblestones
	{Steeples|Scaffolds}
	

streetprops
	Nets, Ropes
	Crates, Boxes
	Cables, Chains
	Drain Pipes
	Water Pumps
	Oil Drums
	Brick Piles
	Iron Bars
	Wooden Boards
	Cut Stones
	Loose Rocks
	Cement Buckets
	a Sewer Grate
	Rotting Refuse
	Mud Puddles
	Discarded Junk
	Carrion & Crows
	Sodden Trash
	Carriages
	Push Carts
	Moored Boats
	a Cargo Barge
	Gondolas
	Wagons
	Cranes & Pulleys
	Cargo Bales
	Metal Ingots
	an Industrial Forge
	{Coal|Fuel}
	Waste Bins
	Street Lamps
	Electric Wires
	Junction Boxes
	a Spotlight Tower
	a Clock Tower
	a Messenger Post
	Withered Trees
	a Monument
	a Fountain
	a Mossy Ruin
	a Collapsed Building
	a Flimsy Hovel
	a Barricade
	a Gate
	a Checkpoint
	Piled Rubble
	a Canal Lock
	a Lightning Barrier
	a Food Stall
	a Vendor Stall
	{Barrels|Casks}
	a Makeshift Shrine
	a News Stand
	a Stockade
	{an Alley|a Sewer Grate}


streetsmells
	{Cook Fires|Roasting Eel|Mushroom soup}
	Furnaces
	Damp Wood
	Trash
	Animals
	Hides
	Blood
	Chemicals 
	Fumes
	{Rain Water|Ocean Water}
	Sewage
	Ozone (Electroplasmic Discharges)


streetsounds
	Clanking Machinery
	Workers 
	Fluttering Cloth 
	Howling Wind
	Laughter
	Songs
	Music 
	{Whispers|Echoes|Strange Voices}
	{Thunder|Driving Rain} 
	{Bells|Clock Chimes|Harbor Horns}
	
	
buildingusecommon
	Bunk House 
	Inn 
	Tavern 
	Gambling Hall 
	Drug Den 
	Brothel
	Market 
	Workshop 
	Bakery 
	Butchery 
	Forge 
	Tailory
	Work House 
	Goat Stables 
	Brewery 
	Watch Post 
	{Court|Jail} 
	Dock
	Ruin 
	Row Houses 
	Tenements
	Apartment Building
	Small House 
	Bath House
	Shrine 
	Tattoo Parlor 
	Physicker Office
	Fighting Pit
	Warehouse 
	Stockyard 
	Factory 
	Refinery 
	Eelery


buildinguserare
	Market House 
	Restaurant 
	{Bar|Lounge} 
	Academy 
	Salon 
	Cafe
	Floristry 
	Tobacconist 
	Book Shop 
	Jeweler 
	Clothier 
	Gallery
	Apothecary 
	Horse Stables 
	Distillery 
	Vintner 
	Boat House
	Theater Opera House
	Apartment	Building
	Townhouse 
	Manor House 
	Villa
	Clinic 
	Temple  
	Archive 
	Church
	Bank 
	Alchemist Lab
	Power Plant

buildingdetails
	Dripping Water
	Creaking	Floorboards
	Roaring Fires 
	Smoky Lamps
	Buzzing Electric Lights
	Ticking	Clockworks
	{Plants|Flowers}
	{Wall Hangings|Artwork}
	Shuttered Windows
	{Heavy Curtains|Thick Carpet}
	Dust 
	Damage
	Utilitarian Furnishings
	Elegant Finery
	Rough-Spun Simplicity
	Spartan Austerity
	{Circular Stairs|Ladders}
	Secret Doors 
	Catwalks 
	Skylights 
	Balcony 
	Cellar
	Strange Devices
	Weird Artifacts
	{Spirit Wards|Old Runes}
	Piled Jumble of Curios
	Antique Appointments
	{Shrine|Altar}


MRbuildingfeatures
	Basement access
	Bright lighting
	Broken furniture
	Broken glass
	Cabinets
	Carpeted floors
	Chandeliers
	Drain pips
	Dumbwaiters
	Marble floors
	Hanging chains
	Huge fireplace
	Open windows
	Decratively showcased Weapons
	Patrols
	Piles of trash
	Pillars
	Rotting ceiling
	Rotting floors
	Rotting walls
	Screens
	Servant passages
	Sewer access
	Shadowy alcoves
	Skylights
	Spyholes
	Staircases
	Tall bookshelves
	Watchdogs
	Window drapes


buildigndescription
	Threadbare
	Tattered
	Lush 
	Comfortable
	Drafty 
	Cold 
	Stout 
	Quiet 
	Cozy 
	Warm
	Vaulted  
	Spacious
	Low 
	Cramped
	Rickety
	Ramshackle
	
exteriormaterial
	Gray Brick
	Timbers
	Cut Stone Blocks
	Wooden Boards
	Plaster Board
	Metal	Sheeting
	
exteriordetails
	Tile
	Iron
	Glass
	Stone
	Wood 
	Landscaping

adjective
	dilapidated
	inferior
	poor
	cheap
	bad
	average
	good
	great
	excellent
	superb


buildingprops
	[k = buildingitems.consumableList], [k], [k], and [k]
	[ti = MRtoolitems.consumableList], [ti], [ti], and [ti]


MRtoolitems
	Acid flask
	Animal trap
	Bellows
	Bolt-cutters
	Chain
	Chisel
	Crowbar
	Door ram
	Ear trumpet
	Fire oil
	Fishing hook
	Googles
	Grappling hook
	Grease
	Hacksaw
	Hammer
	Hand drill
	Lantern
	Lens
	Lock/key
	Lockpics
	Manacles
	Metal file
	Mortar/pestle
	Needle
	Pickaxe
	Pitchfork
	Pliers
	Poles
	Pulleys
	Rope
	Scissors
	Shovel
	Spikes
	Steel wire
	Tongs
	

buildingitems
	a Chalkboard
	Desks 
	Papers
	Maps 
	Charts
	Diagrams
	Books
	Scrolls
	Bookcases
	Lamps
	an Inkwell
	a Writing Desk
	a Clock 
	a Cabinet
	Shelves
	a Table 
	Chairs
	Notebooks
	a Bed 
	a Bureau
	a Vanity
	Bunks 
	Stools
	Trunks
	a Basin 
	a Pitcher
	a Mirror
	a Sofa 
	a Divan
	a Music Box
	Couches
	a Table 
	Drapery 
	Pillows
	Cushions
	a Counter 
	a Sink
	Cabinets
	a Cookfire 
	Pots
	Pans 
	Utensils
	a Dining Table
	Chairs 
	Cutlery
	a Game Board
	Cards 
	Dice
	a Larder
	Spices
	Meat Hooks
	Wine 
	Beer
	Whiskey
	a Pedestal 
	a Statue
	Paintings
	a Bird Cage
	a Quill 
	a Diary
	a Bell 
	a Book
	a Candle
	a Fireplace 
	a Rug
	an Armchair
	Curtains
	Vases 
	Flowers
	Instruments
	Music Sheets
	an Exam Chair
	Medical Tools
	a Burner 
	Vials
	Beakers
	a Workbench
	Tools 
	Rags
	Weapons
	Ammunition	
```

**Comment by cristianmtr** [comment]
```
1 Assassins
1 Accident A killing with no telltale sign of murder.
2 Disappearance The victim vanishes without a trace.
3 Murder An obvious killing to send a message.
4 Ransom A kidnapping and demand for payment for their return.

2 Bravos
1 Battle Defeat an enemy with overwhelming force.
2 Extortion Threaten violence unless you’re paid off.
3 Sabotage Hurt an opponent by destroying something.
4 Smash & Grab A fast and violent armed robbery.

3 Cult

1 Acquisition Procure an arcane artifact and attune it to your god.
2 Augury Do what you must to attract the god’s attention and counsel.
3 Consecration Anoint a place for your deity.
4 Sacrifice Destroy what is valuable or good in honor of your god.

4 Hawkers

1 Sale A significant transaction with a special buyer of illicit product.
2 Supply A transaction to acquire new product or establish a fresh supplier.
3 Show of Force Make an example of an enemy to dominate territory.
4 Socialize Improve customer and/or supplier relations with a social event.

5 Shadows
1 Burglary Theft by breaking and entering.
2 Espionage Obtain secret information by covert or clandestine means.
3 Robbery Theft by force or threats.
4 Sabotage Hurt an opponent by destroying something.

6 Smugglers
1 Smuggle Arcane/Weird Spirit essences, ghosts, cult materials.
2 Smuggle Arms Restrict military weapons, heavy ordnance, explosives.
3 Smuggle Contraband High-tax luxuries, drugs, banned art, etc.
4 Smuggle Passengers People or livestock traveling in secret. 
```

**Comment by cristianmtr** [comment]
Closed by #17 , #16 and #15
